src/main.rs

```diff
@@ -88,18 +88,16 @@ trait RunCommand {
     fn run_command(&mut self) -> Result<()>;
 }
 
-#[derive(Clone, Debug, StructOpt)]
-#[structopt(
-    setting(structopt::clap::AppSettings::SubcommandRequiredElseHelp),
-    setting(structopt::clap::AppSettings::GlobalVersion),
-    version(option_env!("CARGO_PKG_VERSION").unwrap_or("0.0.0")),
-    about(option_env!("CARGO_PKG_DESCRIPTION").unwrap_or("")),
-    // Cargo passes in the subcommand name to the invoked executable. Use a
-    // hidden, optional positional argument to deal with it.
```

Comment:
Let's keep this comment explaining why we have a "dummy" argument.

Reply:
Reverted